index.ts

```diff
@@ -7,30 +20,54 @@ const broccoliFeatures = Object.freeze({
   volatileFlag: true,
 });
 
-function isPossibleNode(node) {
+function isPossibleNode(node: BroccoliNode | string): boolean {
   let type = typeof node;
 
   if (node === null) {
     return false;
```

Comment:
we can convert this to a user-defined type guard

```ts
// eslint-disable-next-line @typescript-eslint/no-explicit-any
function isPossibleNode(node: any): node is BroccoliNode | string {
```

so that we can narrow w/ a condition

```ts
let x: any;
// x is any
if (isPossibleNode(x)) {
   // x is BroccoliNode | string
}
```